.eslintrc.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   env: {
     browser: true,
     es2021: true,
+    node: true,
```

Comment:
Fixes lint errors within the config files — could otherwise ignore them but this is just as easy

Reply:
Instead of doing this for the entire project, add this comment at the top of the config files:
`/* eslint-env node */`